undebt/cmd/main.py

```diff
@@ -102,23 +83,6 @@ def _find_files(paths, extensions):
         if os.path.isfile(path):
             yield path
 
-        else:
```

Comment:
Alternatively, we could remove `_find_files`, use `args.files` and then let it error out if something else (like a directory) gets passed in. I imagine that would cause an error to happen somewhere else in the code.